libraries/Microsoft.Bot.Builder.Ai.Translation/LocaleConverterMiddleware.cs

```diff
@@ -1,94 +1,75 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+
```

Comment:
* XML Comments in many files seems to conflict with recent changes made by @JonathanFingold. We should preserve the work he did, as the code comments get turned into the docs.